integrations/cdi/narayana-jta-cdi/src/main/java/io/helidon/integrations/narayana/jta/cdi/NarayanaTransactionManager.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.integrations.narayana.jta.cdi;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.Destroyed;
+import javax.enterprise.context.Initialized;
+import javax.enterprise.event.Event;
+import javax.inject.Inject;
+import javax.transaction.HeuristicMixedException;
+import javax.transaction.HeuristicRollbackException;
+import javax.transaction.NotSupportedException;
+import javax.transaction.RollbackException;
+import javax.transaction.SystemException;
+import javax.transaction.Transaction;
+import javax.transaction.TransactionManager; // for javadoc only
+import javax.transaction.TransactionScoped;
+
+import com.arjuna.ats.jta.common.JTAEnvironmentBean;
+
+/**
+ * A {@link DelegatingTransactionManager} in {@linkplain
+ * ApplicationScoped application scope} that uses the return value
+ * that results from invoking the {@link
+ * JTAEnvironmentBean#getTransactionManager()} method as its backing
+ * implementation.
+ *
+ * @author <a href="mailto:laird.nelson@oracle.com">Laird Nelson</a>
+ *
+ * @see com.arjuna.ats.jta.common.JTAEnvironmentBean#getTransactionManager()
+ */
+@ApplicationScoped
+class NarayanaTransactionManager extends DelegatingTransactionManager {
+
+  private final Event<Transaction> transactionScopeInitializedBroadcaster;
```

Comment:
This file seems to be indented with 2 spaces instead of 4.

Reply:
Done.